app/src/main/java/com/droidhats/campuscompass/views/FloorFragment.kt

```diff
@@ -0,0 +1,112 @@
+package com.droidhats.campuscompass.views
+
+import android.app.Activity
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.drawable.BitmapDrawable
+import android.os.Bundle
+import android.util.DisplayMetrics
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ToggleButton
+import androidx.core.view.GravityCompat
+import androidx.drawerlayout.widget.DrawerLayout
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProviders
+import androidx.navigation.fragment.findNavController
+import com.droidhats.mapprocessor.ProcessMap
+import com.caverock.androidsvg.SVG
+import com.droidhats.campuscompass.R
+import com.droidhats.campuscompass.viewmodels.FloorViewModel
+import com.mancj.materialsearchbar.MaterialSearchBar
+import com.otaliastudios.zoom.ZoomImageView
+import kotlinx.android.synthetic.main.search_bar_layout.*
```

Comment:
Please don't use * when importing packages.
We have discussed it's better to specify the required components.